src/app/playground/page.tsx

```diff
@@ -49,6 +50,15 @@ const PlaygroundPage: React.FC = () => {
           icon={<ArrowRightIcon />}
         />
       </div>
+      <div>
+        {/* PrimaryButton */}
+        <PrimaryButton
+          text="Sign in"
+          onClick={() => {
+            emptyFunction();
```

Comment:
```suggestion
            //
```
No need to declare an empty function, just add a blank comment line